.travis.yml

```diff
@@ -3,29 +3,35 @@ dist: xenial
 python:
 - 3.6
 install:
-- pip install pipenv
-- pipenv install --dev --deploy --python `which python`
-- pipenv graph
-script: pipenv run py.test -v
+- curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python -
+- source $HOME/.poetry/env
+- which poetry
+- poetry install -vv
+- poetry show
+
+script:
+- poetry run pytest -vv ./test
+after_script:
+- pip install codacy-coverage
+- python-codacy-coverage -r coverage.xml
+
 before_deploy:
-- sed -i.bak "s/=get_version()/='$TRAVIS_TAG'/g" setup.py
-- cat setup.py
-- if [ $(python setup.py --version) == '0.0.0' ]; then travis_terminate 1; fi
+- poetry config http-basic.pypi $PYPI_USER $PYPI_PASSWORD
```

Comment:
`PYPI_USER` and `PYPI_PASSWORD` need to be added to the encrypted travis env vars

Reply:
thanks